docs/DeveloperGuide.adoc

```diff
@@ -242,9 +265,56 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+// tag::sorttransactions[]
+=== Sort Transactions Feature: `sort`
+
+This feature allows the user to _sort_ their bank account by the _amount_ or _date_ of their transactions by using the command `sort amount` or `sort date` respectively.
+
+==== Current Implementation
+
+The `sort` command is facilitated by the _Logic_ and _Model_ components of the application, _PalPay_.
+
+The following sequence diagram shows how the sorting of transactions work when the user enters `sort amount`.
+
+image::SortSequenceDiagram.png[]
+Figure [Fill in Later].
+Sequence Diagram for `sort amount`
+
+-To Be Continued-
+
+// end::sorttransactions[]
+
+// tag::split[]
+=== Split Feature: `split`
+
+This feature allows the user to pay for a certain item or make a transaction on behalf of his friends.
+Refer to the <<UserGuide#Features, UserGuide>> for usage details.
+
+==== Current Implementation
+
+The `split` command is an abstraction of `LendMoney` class. +
+Given a list of *shares* and *people*, each person is assigned an *amount* based on the corresponding positional share and
+the total amount given to `split` command. +
+A `LendMoney` instance is created for each person and executed.
+
+// TODO: add PUML figure here
+
+==== Design Considerations
+
+Current implementation of `Split` class encourages code reuse by abstracting the delegating the task of rebalancing to another class. +
+However, this introduces coupling as the behavior of `Split` is now inexplicably tied to `LendMoney`.
+
+// end::split[]
+
+=== Settle Up Feature: `pay`
+
+This feature allows the user to send money to another person. +
+The balance in the `Ledger` and the balance of the recipient is updated accordingly.
+
```

Comment:
What is the definition of Settle Up Feature? 
Consider adding more details on this feature, for e.g. activity diagram.